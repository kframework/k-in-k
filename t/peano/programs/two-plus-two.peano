plus(succ(zero),succ(zero))
