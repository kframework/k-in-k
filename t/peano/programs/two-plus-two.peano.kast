plus { } ( succ { } ( succ { } ( zero { } ( ) ) ) , succ { } ( succ { } ( zero { } ( ) ) ) )
