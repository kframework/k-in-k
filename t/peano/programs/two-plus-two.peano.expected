succ{}(succ{}(succ{}(succ{}(zero{}()))))
